microservices.yml

```diff
@@ -391,6 +392,51 @@ commands:
             # run dangerfile
             ./node_modules/.bin/danger ci --failOnErrors
 
+  publish-unit-test-results-to-xray:
+    description: Publish all unit test results to Jira Xray
+    steps:
+      - run:
+          name: Download publish to Xray script
+          when: always
+          command: |
+           # get push-test-results-to-xray.sh
+            wget https://raw.githubusercontent.com/MGMDV-Orbs/microservices/f/xray-r2.3/.circleci/push-test-results-to-xray.sh -O push-test-results-to-xray.sh
```

Comment:
Since the script is just bash you can just have it in this file and use circleci parameters instead rather than a separate one and fetching that file manually.